sdk/identity/azure-identity/README.md

```diff
@@ -77,8 +71,6 @@ the Azure Cloud because it combines common production credentials with
 development credentials. `DefaultAzureCredential` attempts to authenticate via
 the following mechanisms in this order, stopping when one succeeds:
 
-![DefaultAzureCredential authentication flow](https://raw.githubusercontent.com/Azure/azure-sdk-for-python/main/sdk/identity/azure-identity/images/DefaultAzureCredentialAuthenticationFlow.png)
```

Comment:
I imagined we'd keep this one, it being notably useful and not a screenshot of a product we don't control?

Reply:
Agreed. I think this is really useful and I would recommend other languages to add it if they don't have it.

Reply:
Makes sense to me -- I'll add it back